README.md

```diff
@@ -173,6 +173,33 @@ const styles = StyleSheet.create({
 
 Aphrodite will ensure that the global `@font-face` rule for this font is only inserted once, no matter how many times it's referenced.
 
+### Caveats
```

Comment:
I think that the Heading that should be used here is probably equal to the other headings, so a single `#` is probably good.